tests/foreman/ui_airgun/test_host.py

```diff
@@ -134,6 +145,248 @@ def module_host_template(module_org, module_loc):
     return host_template
 
 
+@pytest.fixture(scope='module')
+def default_partition_table():
+    # Get the Partition table ID
+    return entities.PartitionTable().search(
+        query={u'search': u'name="{0}"'.format(DEFAULT_PTABLE)})[0]
+
+
+@pytest.fixture(scope='module')
+def default_architecture():
+    # Get the architecture ID
+    return entities.Architecture().search(
+        query={u'search': u'name="{0}"'.format(DEFAULT_ARCHITECTURE)})[0]
+
+
+@pytest.fixture(scope='module')
+def module_libvirt_environment(module_org, module_loc):
+    # Search for puppet environment and associate location
+    environment = entities.Environment(organization=[module_org]).search()[0]
```

Comment:
I'am not sure that this will return the environment related to organization (it can return all the environments) 
to be more sure use
```python
entities.Environment().search(query=dict(search="organization_id={}".format(module_org.id)))[0]
```

Reply:
According to:
https://nailgun.readthedocs.io/en/latest/examples.html#search

>If any attributes have been set, they are used. This finds all lifecycle environments that have a name of “foo” and that belong to organization 1:
`lc_envs = LifecycleEnvironment(name='foo', organization=1).search()`